apps/web/modules/ee/whitelabel/email-customization/lib/organization.ts

```
@@ -0,0 +1,111 @@
+
+    return true;
+  } catch (error) {
+    if (error instanceof Prisma.PrismaClientKnownRequestError && error.code === "P2016") {
+      throw new ResourceNotFoundError("Organization", organizationId);
+    }
+
+    throw error;
+  }
+};
+
+export const removeOrganizationEmailLogoUrl = async (organizationId: string): Promise<Boolean> => {
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)